backend/tests/unit/utils/test_template_resolver_env_file.py

```
@@ -0,0 +1,444 @@
+"""
+Tests for .env file functionality in TemplateResolver.
+
+This module tests the new .env file loading and priority order functionality
+added to the template resolution system.
+"""
+
+import os
+import tempfile
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)